Src/Coravel.Mailer/Mail/Mailers/SmtpMailer.cs

```diff
@@ -121,14 +126,19 @@ private static void SetRecipients(IEnumerable<MailRecipient> to, MimeMessage mai
 
         private void SetFrom(MailRecipient @from, MimeMessage mail)
         {
-            mail.From.Add(AsMailboxAddress(this._globalFrom ?? @from));
+            mail.From.Add(AsMailboxAddress(@from ?? this._globalFrom));
```

Comment:
This was a change from one of your previous issues that you ok'ed and requested them to create a PR for this work. They did not, so I added it to this one. As we as well were very confused on why when we manually set the from address in a single mail, it would always use the global default. I mentioned this in the PR notes. `Have From() override default global From. #321`

Reply:
I would also debate that you cannot use this feature without this change, because if you apply the global from, you can never set the from in the mail.